API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -58,17 +48,25 @@ A simple spatial filter can be supplied as a bounding box. This is a common way
   </p>
   <h4 class="rulelab">How to test</h4>
   <ul>
-    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter.</li>
-    <li>Validate that a document was returned with a status code 200.</li>
+    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter and <code>bbox-crs</code> header containing a CRS supported by the provider. If the provider supports CRS84 then the <code>bbox-crs</code> header may be omitted.</li>
```

Comment:
> If the provider supports CRS84

Dit is verplicht als de API OGC API Features compliant is. Ik zou dit zinnetje hier weg laten, is verwarrend.